lib/config/themes.dart

```diff
@@ -128,6 +128,9 @@ abstract class FluffyThemes {
           onSecondary: brightness == Brightness.light
               ? AppColor.secondaryPrimary
               : AppColor.colorDEEEFF,
+          onPrimaryContainer: brightness == Brightness.light
+              ? Colors.white
+              : const Color.fromARGB(255, 25, 25, 26)
```

Comment:
You should convert color to same format, the heximal form

Reply:
I can't use heximal, flutter only accepts integer in parameter I can convert it to 32 bits if you prefer

Reply:
done it